src/styles/components/_card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin-bottom: 0;
+    margin-top: 40px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    .uppercase {
+      text-transform: uppercase;
+    }
+  }
+
+  &__product-code {
+    margin-bottom: 0;
+    margin-top: 4px;
+
+    line-height: 14px;
+    color: getColor("secondary-text");
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+
+    &-count {
+      margin: 0;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 0;
+    margin-top: 24px;
+
+    &-title,
+    &-value {
+      margin: 0;
+    }
+
+    &-title {
+      font-size: 12px;
+      line-height: 18px;
+      color: getColor("secondary-text");
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button-buy {
+    display: block;
+    text-decoration: none;
+    text-align: center;
+    text-transform: uppercase;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    margin-top: 16px;
+
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    background-color: getColor("accent-button");
+    border: 1px solid transparent;
+
+    &:hover {
+      color: getColor("accent-button");
+
+      background-color: #fff;
+      border: 1px solid getColor("accent-button");
```

Comment:
9'th point in checklist

Reply:
Thanks, I don't understand this point by first, I change it.